tests/test_catalog.py

```diff
@@ -1467,3 +1468,11 @@ def test_remove_hierarchical_links(label_catalog: Catalog, add_canonical: bool)
     for link in label_catalog.links:
         assert not link.is_hierarchical()
     assert bool(label_catalog.get_single_link("canonical")) == add_canonical
+
+
+def test_fully_resolve(tmp_path: Path, label_catalog: Catalog) -> None:
```

Comment:
Is there a naming mixup in the fixtures? `label_catalog` in `conftestp.py` is pulling from `TestCases.case1`, which is not a `label_catalog`. `TestCases` `case4` and `case7` look to be the `label_catalog`.

Reply:
Hm, there are labels in `case1` as well, so that's why I named it that -- I didn't look all the way to `case4` and `case7`. Do you have a better name for `label_catalog`? Now would be a great time to get 'em right :-).

Reply:
I don't see a clean solution. I was thinking to use the catalog id, but there those are not great (e.g., the test-case-1 id is "test"). While not descriptive of the contents, the least confusing may be to name them per their directory for something close to a 1:1 map:

- `cbers_partial_catalog`
- `label_v081_catalog`
- `planet_example_v100_beta2_catalog`
- `test_case_1_catalog`
- `test_case_2_catalog`
- ...

That's kind of hideous, but at least you know where the fixture data comes from.

Reply:
Yeah, that's probably the best option given the current setup... I would love to give the fixtures more descriptive names, but that may require more work /discovery about what the fixtures are actually doing and why they're different from each other.

In this PR, I'll change the fixtures to your recommendation, and then I'll also open an issue to give the fixtures more descriptive names.

Reply:
Added a comment re: meaningful names: https://github.com/stac-utils/pystac/issues/948#issuecomment-1442466189.